rmgpy/statmech/vibrationTest.py

```diff
@@ -32,183 +32,188 @@
+
     def test_getEntropy_quantum(self):
         """
         Test the HarmonicOscillator.getEntropy() method using a set of quantum
         oscillators.
         """
         self.mode.quantum = True
-        Tlist = numpy.array([300,500,1000,1500,2000])
-        Sexplist = numpy.array([0.384065, 0.969182, 2.29837, 3.31251, 4.09675]) * constants.R
-        for T, Sexp in zip(Tlist, Sexplist):
-            Sact = self.mode.getEntropy(T)
-            self.assertAlmostEqual(Sexp, Sact, delta=1e-4*Sexp)
+        t_list = np.array([300, 500, 1000, 1500, 2000])
+        s_exp_list = np.array([0.384065, 0.969182, 2.29837, 3.31251, 4.09675]) * constants.R
+        for temperature, s_exp in zip(t_list, s_exp_list):
+            s_act = self.mode.getEntropy(temperature)
+            self.assertAlmostEqual(s_exp, s_act, delta=1e-4 * s_exp)
 
     def test_getSumOfStates_classical(self):
         """
-        Test the HarmonicOscillator.getSumOfStates() method using a set of 
+        Test the HarmonicOscillator.getSumOfStates() method using a set of
         classical oscillators.
         """
         self.mode.quantum = False
-        self.mode.frequencies = ([500, 1000],"cm^-1")
-        Elist = numpy.arange(0, 10000*11.96, 1*11.96)
-        sumStates = self.mode.getSumOfStates(Elist)
-        densStates = self.mode.getDensityOfStates(Elist)
-        for n in range(10, len(Elist)):
-            self.assertTrue(0.8 < numpy.sum(densStates[0:n]) / sumStates[n] < 1.25, '{0} != {1}'.format(numpy.sum(densStates[0:n]), sumStates[n]))
-        
+        self.mode.frequencies = ([500, 1000], 'cm^-1')
+        e_list = np.arange(0, 10000 * 11.96, 1 * 11.96)
+        sum_states = self.mode.getSumOfStates(e_list)
+        dens_states = self.mode.getDensityOfStates(e_list)
+        for n in range(10, len(e_list)):
+            self.assertTrue(0.8 < np.sum(dens_states[0: n]) / sum_states[n] < 1.25,
+                             '{0} != {1}'.format(np.sum(dens_states[0: n]), sum_states[n]))
+
     def test_getSumOfStates_quantum(self):
         """
-        Test the HarmonicOscillator.getSumOfStates() method using a set of 
+        Test the HarmonicOscillator.getSumOfStates() method using a set of
         quantum oscillators.
         """
         self.mode.quantum = True
-        Elist = numpy.arange(0, 10000*11.96, 1*11.96)
-        sumStates = self.mode.getSumOfStates(Elist)
-        densStates = self.mode.getDensityOfStates(Elist)
-        for n in range(1, len(Elist)):
-            if sumStates[n-1] == 0:
-                self.assertTrue(numpy.sum(densStates[0:n]) == 0, '{0} != {1}'.format(numpy.sum(densStates[0:n]), 0))
+        e_list = np.arange(0, 10000 * 11.96, 1 * 11.96)
+        sum_states = self.mode.getSumOfStates(e_list)
+        dens_states = self.mode.getDensityOfStates(e_list)
+        for n in range(1, len(e_list)):
+            if sum_states[n - 1] == 0:
+                self.assertTrue(np.sum(dens_states[0: n]) == 0,
```

Comment:
This test seems strange. Perhaps lets do `assertFalse(expression)` instead of `assertTrue(expression == 0)`.
Also, the message (`'{0} != {1}'`) is trivial, we can get rid of it.

Reply:
I think `assertFalse` might be more general than necessary. It should be changed to `assertEqual` though, which has that as the default failure message.

Reply:
Now using `assertEqual` without extra message. I also remove trivial message for some `assertAlmostEqual`